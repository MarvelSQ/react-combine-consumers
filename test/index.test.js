import React from 'react';
import renderer from 'react-test-renderer';
import App from './App';

test('test', () => {
  expect(1 + 1).toBe(2);
});

test('render', () => {
  const component = renderer.create(<App />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

  tree.props.onChangeColor('#2196f3');

  tree = component.toJSON();
  expect(tree).toMatchSnapshot();

  tree.props.onChangeHeight('30px');

  tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

