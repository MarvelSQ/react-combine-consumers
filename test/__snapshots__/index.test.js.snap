// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render 1`] = `
<div
  onChangeColor={[Function]}
  onChangeHeight={[Function]}
>
  <div
    style={
      Object {
        "color": "#ff00ff",
        "height": "40px",
      }
    }
  >
    #ff00ff
    ,
    40px
  </div>
  <button
    style={
      Object {
        "color": "#ff00ff",
        "height": "40px",
      }
    }
  >
    40px
    ,
    #ff00ff
  </button>
</div>
`;

exports[`render 2`] = `
<div
  onChangeColor={[Function]}
  onChangeHeight={[Function]}
>
  <div
    style={
      Object {
        "color": "#2196f3",
        "height": "40px",
      }
    }
  >
    #2196f3
    ,
    40px
  </div>
  <button
    style={
      Object {
        "color": "#2196f3",
        "height": "40px",
      }
    }
  >
    40px
    ,
    #2196f3
  </button>
</div>
`;

exports[`render 3`] = `
<div
  onChangeColor={[Function]}
  onChangeHeight={[Function]}
>
  <div
    style={
      Object {
        "color": "#2196f3",
        "height": "30px",
      }
    }
  >
    #2196f3
    ,
    30px
  </div>
  <button
    style={
      Object {
        "color": "#2196f3",
        "height": "30px",
      }
    }
  >
    30px
    ,
    #2196f3
  </button>
</div>
`;
